kuksa-sdk/src/test/kotlin/org/eclipse/kuksa/extensions/DataBrokerApiInteractionExtensions.kt

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package org.eclipse.kuksa.extensions
+
+import io.kotest.assertions.fail
+import org.eclipse.kuksa.DataBrokerApiInteraction
+import org.eclipse.kuksa.proto.v1.Types
+import kotlin.random.Random
+
+internal suspend fun DataBrokerApiInteraction.setRandomFloatValue(vssPath: String, maxValue: Int = 300): Float {
```

Comment:
maybe a name change makes sense here, "set" does not really reflect that we are "updating the value on the DataBroker"